packages/vscode/src/extension.ts

```diff
@@ -102,13 +104,24 @@ async function addWorkspaceFolder(
             ) {
                 const originalUri = document.uri.toString(true);
 
-                const isInsideHtmlRegionCommand = `${Command.IsInsideHtmlRegion}(${workspaceUri})`;
                 const isInsideHtmlRegion = await commands.executeCommand<boolean>(
-                    isInsideHtmlRegionCommand,
+                    Command.IsInsideHtmlRegion,
+                    workspaceFolder.uri.fsPath,
                     originalUri,
                     position,
                 );
-                const result = await unwrapCompletionArray(next(document, position, context, token)) as ProtocolCompletionItem[];
+                const completions = next(document, position, context, token);
+                const result = (await unwrapCompletionArray(completions) as ProtocolCompletionItem[])
+                    .map(completion => {
+                        const [ insertStart, insertEnd ] = completion.data as [ number, number ];
```

Comment:
This breaks completion for variables. Repro:

```twig
{% set variable = '123' %}

{{ | }}
```

`completion.data` is undefined in this case

Reply:
At `template-paths.ts`, apply the `textEdit` fix. It will create the exact same object with the `range` property, without the need to map it in the `extension.ts` file